websites/M/MultiplayerPiano.net/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "224864947480297472",
+		"name": "antokek"
+	},
+	"service": "MultiplayerPiano.net",
+	"description": {
+		"en": "An online website used for playing piano with friends. It supports input from both MIDI keyboards and QWERTY keyboards."
```

Official description.
```suggestion
		"en": "An online piano you can play alone or with others in real-time. MIDI support, 88 keys, velocity sensitive. You can show off your skill or chat while listening to others play."
```